<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{background-color:#fff9a2;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>531</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color:#bbfa62;
border: 2px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineaDNI">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:0px}</string>
     </property>
     <property name="text">
      <string>Cliente N</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineaCIudad">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineaDetalle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:0px}</string>
     </property>
     <property name="text">
      <string>Detalle</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:0px}</string>
     </property>
     <property name="text">
      <string>Direccion</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:0px}</string>
     </property>
     <property name="text">
      <string>Ciudad</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:0px}</string>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineaNombre">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{border:0px}</string>
     </property>
     <property name="text">
      <string>DNI/CIF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineaDireccion">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonBorrar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAtras">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>422</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAceptar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBorrar</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>borrarCliente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAtras</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>lanzarListarClientes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAceptar</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>lanzarModificarCliente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>borrarCliente()</slot>
  <slot>lanzarListarClientes()</slot>
  <slot>lanzarModificarCliente()</slot>
 </slots>
</ui>
